templates/lb-fargate-service/cf.yml

```diff
@@ -1,168 +1,174 @@
 # Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
-
-# TODO: handle conditional Load Balancer in environment
-# TODO: Parameterize TaskDefinition/Service constructs
-
+AWSTemplateFormatVersion: 2010-09-09
+Description: CloudFormation template that represents an load balanced web application.
```

Comment:
that represents a load balanced fargate web application on ECS

Reply:
Done!